src/chrome/content/rules/5ch.xml

```diff
@@ -0,0 +1,44 @@
+<ruleset name="5ch.net">
+	<target host="5ch.net" />
+	<target host="*.5ch.net" />
+
+	<test url="http://5ch.net/" />
```

Comment:
An implicit test URL http://5ch.net/` is added automatically for non-wildcard `target`. Please also remove this line so Travis will not complain about duplicated test URLs.

Reply:
＠cschanaj Thank you. I was reading the guide, now.😅